src/Dialogs/Plane/PlaneListDialog.cpp

```diff
@@ -234,9 +234,7 @@ PlaneListWidget::NewClicked()
       continue;
     }
 
-    StaticString<42> filename(plane.registration);
-    filename += _T(".xcp");
-
+    StaticString<42> filename( PlaneGlue::GetFileName(plane) );
```

Comment:
Simpler:
```
  const auto filename = PlaneGlue::GetFileName(plane);
```
Using `auto` also allows changing the function's return type (e.g. larger buffer) without changing all the callers.

Reply:
to be done in PR for plane masses